packages/terra-form-textarea/src/Textarea.jsx

```diff
@@ -70,6 +70,11 @@ const propTypes = {
    * The value of the textarea. Use this to create a controlled textarea.
    */
   value: PropTypes.string,
+  /**
+   * Callback to set ref of the textarea. Useful for advanced functionality
+   * such as managing focus, selection, or animations.
+   */
+  setRef: PropTypes.func,
```

Comment:
There's somewhat of a standard for this callback name: `refCallback`.

Here are a few examples:

https://github.com/cerner/terra-core/blob/master/packages/terra-scroll/src/Scroll.jsx#L17
https://github.com/cerner/terra-core/blob/master/packages/terra-list/src/List.jsx#L22
https://github.com/cerner/terra-core/blob/master/packages/terra-tabs/src/_TabMenu.jsx#L25